src/admin/dashboard/components/Header.tsx

```
@@ -1,38 +1,39 @@
-import { useState } from '@wordpress/element';
+import { RawHTML, useState } from '@wordpress/element';
 import HelpIcon from '../icons/HelpIcon';
 import DokanIcon from '../icons/DokanIcon';
 import CrownIcon from '../icons/CrownIcon';
 import IconMapping from './IconMapping';
-import AdminSetupBanner from "../../banner/AdminSetupBanner";
 
 const Header = () => {
     const [ showDropdown, setShowDropdown ] = useState( false );
     const { is_pro_exists, lite_version, help_menu_items } =
         dokanAdminDashboardSettings?.header_info;
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)